src/formpack/reporting/export.py

```diff
@@ -43,13 +44,27 @@ def __init__(self, formpack, form_versions, lang=UNSPECIFIED_TRANSLATION,
             tag_cols_for_header = []
         self.tag_cols_for_header = tag_cols_for_header
 
-        # If some fields need to be arbitrarly copied, add them
+        def snake_case_to_camel_case(str_):
+            """
+            my_variable -> MyVariable
+            :param str_: str
+            :return: str
+            """
+            words = str_.strip("_").split("_")
+            return "".join([word.lower().capitalize() for word in words])
+
+        # If some fields need to be arbitrarily copied, add them
         # to the first section
         if copy_fields:
+            # import schema to load dynamically custom `CopyField` classes such as ValidationStationCopyField
+            schema_module = import_module("formpack.schema")
+
             for version in iter(form_versions.values()):
                 first_section = next(iter(version.sections.values()))
                 for name in copy_fields:
-                    dumb_field = CopyField(name, section=first_section)
+                    class_name = "{}CopyField".format(snake_case_to_camel_case(name))
```

Comment:
Clever, for sure, but I'd really prefer something more explicit. What about passing something like `custom_fields` to the `Export` constructor in addition to `copy_fields`? `custom_fields` could be a list of tuples that each specify a field's name and which class to use for it, e.g.
```python
custom_fields = [
    ('fancy_field_name', 'my.module.FancyField'),
    ('basic_field', 'other.module.NoFrillsField'),
    ...
]
```
This would retain the flexibility of your approach while letting future me `grep` through the project for `ValidationStatusCopyField`.

Importing `my.module.FieldClass` [looks like](https://stackoverflow.com/a/24815361) it can be done with `pydoc.locate`, which _might be_ included with all Python interpreters(?)
![image](https://user-images.githubusercontent.com/2085013/46956372-2d96d700-d063-11e8-8752-dedd7e9e9174.png)

On second thought, why bother with strings? Maybe it would be better (and simpler) to pass classes as the second element of each tuple.

Reply:
Actually, I did test `locate` before using `import_module` and had some issue. 
Anyway, I'm passing the class, so `locate` is not needed anymore.